app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -127,14 +134,16 @@ private fun UserInputtingContent(
         DatePickerDialog(
             context,
             { _: DatePicker, Year: Int, Month: Int, DayOfMonth: Int ->
-                birthDate = "$Year-${Month}-$DayOfMonth"
+                birthDate = "$Year-$Month-$DayOfMonth"
                 val split = birthDate.split("-")
                 val year = split[0].toInt()
                 val month = split[1].toInt()
```

Comment:
You should also `toIntOrNull` just to be safe, it'll crash if invalid

Reply:
added, also added null checks.